.github/workflows/release.yml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Checkout git repo
         uses: actions/checkout@v3
 
-      - uses: ./actions/setup_and_build
+      - name: Setup & Build
+        uses: ./actions/setup_and_build
```

Comment:
I assume you left out "Enforce yarn.lock has no duplicates" here since we should not need it since we never push directly on master 😉

Reply:
indeed :wink: